### PR TITLE
Create template code of conduct

### PR DESCRIPTION
**Pull request by d3zd3z**
Incorporate the [Contributor Covenant](https://www.contributor-covenant.org/)
used by a large number of open source communities.

Signed-off-by: David Brown <david.brown@linaro.org>

### CODE REVIEW

**Review by utzig — APPROVED** [review]

**Review by tamasban — COMMENTED** [review]
Is it intended to be v2.0?

**Comment by d3zd3z** [comment]
> Is it intended to be v2.0?

Yes, isn't it? I'm unclear what your comment is about.

**Comment by tamasban** [comment]
> > Is it intended to be v2.0?
> 
> Yes, isn't it? I'm unclear what your comment is about.

There is a slight difference in the first part compared to this version:
https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md

**Review by agross-korg — APPROVED** [review]

**Comment by nvlsianpu** [comment]
>> > Is it intended to be v2.0?
>> 
>> Yes, isn't it? I'm unclear what your comment is about.

>There is a slight difference in the first part compared to this version:
https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md

Indeed there is the difference.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, religion, or sexual identity
```
Comment (nvlsianpu) [inline CODE_OF_CONDUCT.md:9]: Almost the same, but 2.0 is wider by two words.

**Review by nvlsianpu — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (utzig) [reply CODE_OF_CONDUCT.md:9]: Which are "caste, color", nicely spot.

Comment (d3zd3z) [reply CODE_OF_CONDUCT.md:9]: I suspect that github must have pulled a not-quite-released version of this document for their template. I can update to the correct current version, though.

**Comment by d3zd3z** [comment]
This is the version taken directly from the code of conduct website. In addition to the wording changes mentioned, the footer text is slightly different.

**Review by tamasban — APPROVED** [review]
Thanks for the update!

**Pull request merged**